plugins/repository-s3/src/main/java/org/elasticsearch/cloud/aws/AwsS3Service.java

```diff
@@ -26,6 +26,56 @@
  *
  */
 public interface AwsS3Service extends LifecycleComponent<AwsS3Service> {
+
+    final class CLOUD_AWS {
```

Comment:
We did. But now the project has been split in 2 plugins. So we don't have anymore here `AwsEc2Service`.